Makefile

```diff
@@ -22,13 +22,16 @@ licenser:
 gomod:
 	go mod tidy
 
+test-go:
+	go test -v -count 1 ./...
```

Comment:
Consider adding an explanation for the `-count 1`, similar to https://github.com/elastic/package-spec/blob/e7102f8cbd392d977bb2a21b12674dbe2ae8b985/code/go/Makefile#L33-L35.

Reply:
Fixed